src/SnD.Sdk/Kubernetes/KubernetesApiExtensions.cs

```diff
@@ -503,6 +503,101 @@ public static V1Job WithBillingId(this V1Job job, string billingId)
             });
         }
 
+        /// <summary>
+        /// Merges two V1PodFailurePolicies objects.
+        /// </summary>
+        /// <param name="ruleA"> A V1PodFailurePolicyRule object with the action name and a list of exit codes.</param>
+        /// <param name="ruleB"> A different V1PodFailurePolicyRule object with the action name and a list of exit codes.</param>
+        /// <returns> A V1PodFailurePolicyRule object.</returns>
+        public static V1PodFailurePolicyRule MergePodFailurePolicyRules(
+            V1PodFailurePolicyRule ruleA,
+            V1PodFailurePolicyRule ruleB)
+        {
+            if (ruleA.Action != ruleB.Action)
+            {
+                throw new InvalidOperationException("Cannot merge rules with different actions");
+            }
+
+            if (ruleA.OnPodConditions != ruleB.OnPodConditions)
```

Comment:
You can't compare lists like this

Reply:
I removed this now in [561f1cf](https://github.com/SneaksAndData/esd-services-sdk/pull/75/commits/561f1cf4a699fe803d15ae5b14f721b523caee0e), when originally added I didn't think they were lists but then once I figured it out I didn't updated it.
Thank you for the explanation regarding it 👍